_template/bin/nuget

```diff
@@ -1,2 +1,6 @@
-result = system(File.dirname(__FILE__) + "/nuget.exe " + ARGV.join(' '))
-exit 1 unless result
\ No newline at end of file
+host_os = RbConfig::CONFIG['host_os']
+windows = /mswin|msys|mingw|cygwin|bccwin|wince|emc/.match(host_os)
```

Comment:
jruby reports "mswin32", standard ruby reports "mingw32", cygwin installed reports "cygwin". So I would guess that the other alternatives are not relevant.